src/docs/mise.md

```diff
@@ -0,0 +1,24 @@
+# Mise - Polyglot tool version manager
+
+Mise is a fast, polyglot tool version manager. It can manage versions for a variety of tools like Node.js, Python, Ruby, Go, and more. It's a replacement for tools like `asdf`, `nvm`, and `pyenv`.
+
+## What it does
+
+- Manages multiple versions of development tools on a per-project basis.
+- Uses a `.mise.toml` configuration file to define tool versions.
```

Comment:
can we give a very short example of usage, and link out to official docs for agents to follow